src/components/Person/Person.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import './Person.scss';
+import PropTypes from 'prop-types';
```

Comment:
You should push the package.json (and maybe also lock?) after adding this to dependencies, it should fix the github tests

Reply:
package.json is enough 😅 
![image](https://user-images.githubusercontent.com/60885495/171006186-865b0fc3-8512-4fa1-a5bb-7b2c98d4b6b6.png)